app/metrics.yaml

```diff
@@ -12,13 +12,21 @@ events:
     type: event
```

Comment:
nit: can we get issue numbers for the commits?

Also, git best practices is that a commit summary is 80 characters: that's why github does this wrapping:
![image](https://user-images.githubusercontent.com/759372/90199461-b4b15500-dd89-11ea-92b4-7d659cc7367c.png)

That's pretty short though so I go over a bit if necessary – we could shorten the commit summary and use the extended summary if we wanted to add more detail.